styles/index.css

```diff
@@ -0,0 +1,166 @@
+@import url('https://fonts.googleapis.com/css2?family=PT+Sans+Caption&family=Itim&display=swap');
+body {
+    max-width: 800px;
+    margin: auto;
+    background-color: #ffcac8;
+}
+
+h1,
+h2 {
+    font-family: 'Itim', cursive;
+    text-align: center;
+    color: white;
+    text-shadow: -1px -1px 0 #000, 1px -1px 0 #000, -1px 1px 0 #000, 1px 1px 0 #000;
+    margin: 10px 0px 10px 0px;
+    padding: 0px 0px 0px 0px;
+}
+
+h1 {
+    font-size: 5em;
+    line-height: 0.7em;
+}
+
+h2 {
+    font-size: 3em;
+}
+
+h3 {
+    margin: 10px 0px 10px 0px;
+    padding: 0px 0px 0px 0px;
+    font-family: 'Itim', sans-serif;
+    font-size: 1.5em;
+}
+
+.hidden {
+    border: 0;
+    clip: rect(0 0 0 0);
+    height: 1px;
+    margin: -1px;
+    overflow: hidden;
+    padding: 0;
+    position: absolute;
+    width: 1px;
+}
+
+button {
+    border-radius: 10px;
+    background-color: white;
+    color: #ffcac8;
+    font-family: 'Itim', sans-serif;
+    font-size: 20px;
+    border-width: 0px;
+    width: 40%;
+    margin-top: 10px;
+    margin-left: 10px;
+    margin-right: 10px;
+    padding: 16px 32px;
+    display: inline-block;
+}
+
+button:active {
+    background-color: #ffcac8;
+    color: black;
+}
+
+.button-container {
+    width: 100%;
+    text-align: center;
+}
+
+input[type=text] {
+    width: 100%;
+    border: 1px solid #ffcac8;
+    border-radius: 10px;
+    font-size: 1.25rem;
+    padding-top: 0.5em;
+    padding-bottom: 0.5em;
+    line-height: 1.1;
+    background-color: #fff;
+}
+
+input[type=submit],
+input[type=reset] {
+    background-color: #ffcac8;
+    border: none;
+    border-radius: 10px;
+    color: white;
+    padding: 16px 32px;
+    text-decoration: none;
+    margin: 4px 2px;
+    cursor: pointer;
+    font-family: 'Itim', sans-serif;
+    font-size: 20px;
+    width: 100%
+}
+
+select {
+    appearance: none;
+    background-color: transparent;
+    border: none;
+    padding: 0 1em 0 1em;
+    margin: 2px;
+    width: 100%;
+    font-family: inherit;
+    font-size: inherit;
+    cursor: inherit;
+    line-height: inherit;
+}
+
+.select {
+    width: 100%;
+    border: 1px solid #ffcac8;
+    border-radius: 10px;
+    font-size: 1.25rem;
+    padding-top: 0.5em;
+    padding-bottom: 0.5em;
+    line-height: 1.1;
+    background-color: #fff;
+}
+
+.selection {
+    font-size: 1.5em;
+}
+
+#citySelector,
+#temperatureArea,
+#skySelector {
```

Comment:
We're targeting 3 different ids here, another option would be to give these 3 elements a common class which we target here.